src/providers/supabase/authProvider.ts

```diff
@@ -90,8 +83,26 @@ export const authProvider: AuthProvider = {
 
         return baseAuthProvider.checkAuth(params);
     },
-    async getPermissions(params) {
+    // FIXME: supabaseAuthProvider does not yet provide canAccess
```

Comment:
I don't see how we can provide a generic `canAccess` in `ra-supabase`, so this FIXME should be removed IMHO

Reply:
Like we do for `getPermissions` and `getIdentity`?